doc/modules/cross_validation.rst

```diff
@@ -731,10 +732,11 @@ to shuffle the data indices before splitting them. Note that:
   validation performed by specifying ``cv=some_integer`` to
   :func:`cross_val_score`, grid search, etc. Keep in mind that
   :func:`train_test_split` still returns a random split.
-* The ``random_state`` parameter defaults to ``None``, meaning that the
-  shuffling will be different every time ``KFold(..., shuffle=True)`` is
-  iterated. However, ``GridSearchCV`` will use the same shuffling for each set
-  of parameters validated by a single call to its ``fit`` method.
+* The ``random_state`` parameter defaults to ``None`` and ``shuffle`` to
+  ``False``, meaning that the shuffling will be different every time
```

Comment:
Now the sentence is a bit weird, though... Can you maybe reformulate. Also it would be great if you could add the "see also" mentioned in the issue. Though I'm also happy just merging these additions (but then we shouldn't close the issue)